cmd/workspace/apps/run-local.go

```diff
@@ -0,0 +1,205 @@
+package apps
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/signal"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/libs/appproxy"
+	"github.com/databricks/cli/libs/apps"
+	"github.com/databricks/cli/libs/auth"
+	"github.com/databricks/cli/libs/cmdctx"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/spf13/cobra"
+)
+
+var (
+	port               int
+	debug              bool
+	prepareEnvironment bool
+	entryPoint         string
+	customEnv          []string
+)
+
+func newRunLocal() *cobra.Command {
+	cmd := &cobra.Command{}
+
+	cmd.Use = "run-local"
+	cmd.Short = `Run an app locally`
+	cmd.Long = `Run an app locally.
+
+	  This command starts an app locally.`
+
+	cmd.Flags().IntVar(&port, "port", 8001, "Port on which to run the app")
+	cmd.Flags().BoolVar(&debug, "debug", false, "Enable debug mode")
+	cmd.Flags().BoolVar(&prepareEnvironment, "prepare-environment", false, "Prepares the environment for running the app. Requires 'uv' to be installed.")
+	cmd.Flags().StringSliceVar(&customEnv, "env", nil, "Set environment variables")
+	cmd.Flags().StringVar(&entryPoint, "entry-point", "", "Specify the custom entry point with configuration (.yml file) for the app. Defaults to app.yml")
+
+	cmd.PreRunE = root.MustWorkspaceClient
+	cmd.RunE = func(cmd *cobra.Command, args []string) (err error) {
+		ctx := cmd.Context()
+		w := cmdctx.WorkspaceClient(ctx)
+		workspaceId, err := w.CurrentWorkspaceID(ctx)
+		if err != nil {
+			return err
+		}
+
+		cwd, err := os.Getwd()
```

Comment:
Is this still correct in a bundle context, after the cwd switch change?

Reply:
This needs to use the actual current directory, with the CWD switch this need to be handled separately I believe

Reply:
I think the CWD change is also only focussed on bundle commands and not generally on all CLI commands so unlikely any changes needed here